### ISSUE TITLE
support dynamic properties from index signature

### ISSUE THREAD

**Issue by atsu85** [MeirionHughes/aurelia-template-lint#90]
Currently there is a false positive reported 
(smth like `ERROR: cannot find 'icon' in type 'RouteConfig'`)
when the type(class or interface) allows dynamic properties because of index signature syntax:
```TypeScript
    export interface RouteConfig {
      route: string | string[];
      [x: string]: any; // dynamic properties can be used with index signature
    }
```
and You use smth like
```html
${routeConfig.icon}
```

**Comment by atsu85** [comment]
I'll send a PR shortly
